vnda/utils/transform.ts

```diff
@@ -166,35 +168,93 @@ const toOffer = ({
 const toPropertyValue = (variant: VNDAProduct): PropertyValue[] =>
   Object.values(variant.properties ?? {})
     .filter(Boolean)
-    .map(({ value, name }) =>
-      value && ({
-        "@type": "PropertyValue",
-        name,
-        value,
-        valueReference: "SPECIFICATION",
-      } as PropertyValue)
-    ).filter((x): x is PropertyValue => Boolean(x));
+    .map(
+      ({ value, name }) =>
+        value &&
+        ({
+          "@type": "PropertyValue",
+          name,
+          value,
+          valueReference: "SPECIFICATION",
+        } as PropertyValue),
+    )
+    .filter((x): x is PropertyValue => Boolean(x));
 
 const toPropertyValueTags = (tags: ProductSearch["tags"]): PropertyValue[] =>
-  tags?.map((tag) =>
-    tag && ({
-      "@type": "PropertyValue",
-      name: tag.name,
-      value: JSON.stringify(tag),
-      valueReference: "TAGS",
-    } as PropertyValue)
+  tags?.map(
+    (tag) =>
+      tag &&
+      ({
+        "@type": "PropertyValue",
+        name: tag.name,
+        value: JSON.stringify(tag),
+        valueReference: "TAGS",
+      } as PropertyValue),
   );
 
 // deno-lint-ignore no-explicit-any
 const isProductVariant = (p: any): p is VariantProductSearch =>
   typeof p.id === "number";
 
-const normalizeVariants = (
-  variants: VNDAProductGroup["variants"] = [],
-): VNDAProduct[] =>
-  variants.flatMap((v) =>
-    isProductVariant(v) ? [v] : Object.values(v) as VNDAProduct[]
+// Converts a VNDAProductGroup with no variants into a VNDAProduct
+// *** If VNDAProductGroup has no variants, it is not available ***
+const buildNoVariantsProduct = (productGroup: VNDAProductGroup) => {
+  const {
+    id,
+    image_url = "",
+    installments = [],
+    name = "",
+    price = 0,
+    sale_price = price,
+    slug = "",
+  } = productGroup;
+
+  return {
+    available: false,
+    available_quantity: 0,
+    barcode: "",
+    custom_attributes: {},
+    full_name: name,
+    handling_days: 0,
+    height: 0,
+    image_url,
+    installments,
+    intl_price: 0,
+    length: 0,
+    main: true,
+    min_quantity: 1,
+    name,
+    norder: 0,
+    price,
+    product_id: id!,
+    properties: {},
+    quantity: 0,
+    sale_price,
+    sku: "01",
```

Comment:
Concerning the SKU, considering that using "01" may lead to conflicts, wouldn't it be better to leave it empty `sku: "",` ?